package.json

```diff
@@ -39,47 +26,46 @@
     "jshint": "^2.9.5",
     "minimist": "^1.2.0",
     "mocha": "^3.4.2",
-    "moustache": "0.0.2",
     "npm-run-all": "^4.1.1",
     "nsp": "^2.8.1",
-    "onchange": "^3.2.1",
     "prettier": "^1.10.2",
     "selenium-webdriver": "^3.5.0",
-    "shield-studies-addon-utils": "^4.1.0"
+    "web-ext": "^2.4.0"
   },
   "engines": {
     "node": ">=8.9.0"
   },
   "homepage": "http://github.com/mozilla/shield-studies-addon-template",
   "keywords": [
     "firefox",
-    "legacy-addon",
     "mozilla",
-    "shield-study"
+    "shield-study",
+    "template"
   ],
-  "license": "MIT",
-  "main": "index.js",
+  "license": "MPL-2",
```

Comment:
nit: this should be "MPL-2.0" to be a valid SPDX license.

Currently `npm i` results in the following warning:

```
npm WARN shield-studies-addon-template@2.0.0 license should be a valid SPDX license expression
```